Learning_Guide/Coaching.md

```diff
@@ -0,0 +1,61 @@
+1. **Escalating code blocks and issues to SEPs if you are unable to resolve the learners block during your appointment** (ideally, if after 15 minutes you do not feel you are on a solid path to debugging the issue, a coach should find an available SEP to bring in.)
+
+**Tip**: It is a good practice to block off time at the end and beginning of each day to focus on code reviews, and to keep as much middle-of-the-day time as possible available for coach-queue appointments.
+
+## What relevant skills does coaching teach me?
+
+A software engineer should be able to communicate clearly, ask great questions, help others ask better questions, and find poor code/logic quickly. During a week of coaching, you should focus on the following:
+
+### Context: Coach-queue appointments
+
+- Am I asking high-quality questions? Was I able to communicate without words like: thing, stuff, and this/that/those.
+- Can I quickly establish what I need from the learner to start helping them, and ask for it in a clear way?
+- Can I be confident in a situation where I am expected to help find a solution with little context?
+- How quickly can I determine if I am not able to help in a given situation and how do I respond?
+- Can I communicate with teams who may be having both intrapersonal and technical blocks, and am I able to help coach both programming concepts and better team patterns?
+
+### Context: Code reviews
+
+- Are the comments I leave actionable, specific, and kind?
+- Do I feel comfortable reviewing code when I don’t know 100% of what it is doing?
+- Am I missing common pitfalls when reviewing (i.e. poor indentation, confusing variable naming, not seeing when code could be easily refactored for readability)
+- Am I building confidence in my ability to analyze code that is not mine?
+- Are there major gaps in my own learning that are preventing me from giving a high-quality review and am I tracking those gaps as they are brought to my attention?
+- When I am skilled at a concept am I able to leave refactoring suggestions that consider optimization as well as readability?
+
+### Context: Escalation
+
+- Was I able to identify within 15 minutes if I needed to escalate?
+- Did I feel comfortable admitting I needed to escalate, and am I able to still be confident in following coach-queue appointments after escalating.
+- Was I able to ask high-quality questions of the learner/team so that I could describe to the SEP what the issue is, and where I found myself blocked.
+
+## Additional Tips
+
+- **Always prioritize participating in coaching sessions when you have escalated to an SEP**. This is one of the best opportunities to learn you may encounter. You attempted to solve a problem, and when you could not you get to immediately see how someone with a ton of experience solves the issue. If you have another appointment scheduled move it back so that you get to learn from the SEP.
```

Comment:
**You must stay with the SEP when escalating so you too can learn the solution to the problem**